tensorflow/core/framework/tensor.h

```diff
@@ -35,6 +35,12 @@ limitations under the License.
 #include "tensorflow/core/platform/mem.h"
 #include "tensorflow/core/platform/types.h"
 
+#if !defined(PLUGGABLE_DEVICE_SUPPORTED_MACOS) &&  \
+    defined(__APPLE__) && !defined(ANDROID) &&     \
+    !defined(__ANDROID__) && !TARGET_OS_IOS
```

Comment:
Nit: `!defined(ANDROID)` appeared twice.
```suggestion
    defined(__APPLE__) && !defined(ANDROID) && !TARGET_OS_IOS
```

Reply:
@kulinseth  Sorry I misread ANDROID and __ANDROID__. I thought they were the same (just __ANDROID__ twice). I'll add this back internally!